docs/locales/ko_KR/LC_MESSAGES/common-api.po

```diff
@@ -49,12 +51,15 @@ msgid ""
 "server-side proxy to our API service if you are building a public-facing "
 "front-end service using Backend.AI."
 msgstr ""
+"API 액세스 키와 비밀 키가 외부로 유출되는 것을 막기 위한 보안 조치로, Backend.AI 를 활용해 공개된 "
+"프론트엔드 서비스를 설계할 때는 가급적 API 서비스를 위한 서버사이드 프록시를 사용하는 것을 권장합니다."
 
 #: ../../common-api/auth.rst:21
 msgid ""
 "For local deployments, you may create a master dummy pair in the "
 "configuration (TODO)."
 msgstr ""
```

Comment:
Isn't there a better expression for "dummy" in Korean, instead of "더미"?
There's already a well-known meaning for "더미"(a.k.a piles), so we could to use another word to avoid misunderstanding.